tests/test_browser.py

```diff
@@ -135,12 +135,3 @@ def test_get_cookiejar():
     jar = browser.get_cookiejar()
     assert jar.get('k1') == 'v1'
     assert jar.get('k2') == 'v2'
-
-if __name__ == '__main__':
```

Comment:
I'd like to keep a way to call the script without using the executable `pytest` (just for simplicity, people see a `*.py` file and may want to run `python ./test_file.py` or just `./test_file.py`).

AFAICT, calling `pytest.main(__file__)` would do the trick.

Reply:
That sounds like a fair compromise. I'll give it a try!

Reply:
Actually, `pytest.main(sys.argv)` is better (first, recent pytest warn when you pass a string to main and my initial version didn't allow `./test_foo.py -v` for example).

Search for better option in progress here: https://stackoverflow.com/questions/35353771/invoke-pytest-from-python-for-current-module-only

Reply:
I ran into this suggestion as well, and it appears to provide all the necessary functionality. Thanks!